SCClassLibrary/Common/GUI/PlusGUI/Control/ServerPlusGUI.sc

```diff
@@ -1,33 +1,42 @@
 ServerTreeView {
 	var <server, <bounds, viewParent, <window, resp, updateFunc, updater, tabSize = 25, treeViewStatus;
 
-	*new { |server, bounds=(Rect(128, 64, 400, 400)), parent|
-		if(bounds.asRect.width < 400) {
-			var bnds = bounds.asRect;
-			"The width value you set will be changed to 400, the minimum width.".postln;
-			bounds = Rect(bnds.left, bnds.top, 400, bnds.height);
-		};
+	*new { |server, bounds, parent|
+		server = server ? Server.default;
 		^super.newCopyArgs(server).makeWindow(bounds, parent)
 	}
 
 	makeWindow { |bounds, parent|
-		if(parent == nil) {
-			window = Window.new(this.asString, bounds, scroll:true);
-			viewParent = window;
+		bounds = if(bounds.isNil) {
+			Rect(128, 64, 400, 400)
 		} {
-			if (parent.name.contains(" Node Tree")) {
-				window = parent
+			if(bounds.asRect.width < 400) {
+				var bnds = bounds.asRect;
```

Comment:
if you accept things that can be converted with `asRect`, you should do the conversion befoere the if statement in line 13.


And: `Rect` has `.width` as a method, so `.bounds` is unnecessary.

Reply:
>if you accept things that can be converted with asRect, you should do the conversion befoere the if statement in line 13.

I have moved `var bnds = bounds.asRect;` before `if(bounds.asRect.width < 400) {`.

>And: Rect has .width as a method, so .bounds is unnecessary.

Could you point out the related line?

Reply:
thank you, that was the line I meant.